test/SlowTests/MailingList/TransformWithConversionListener.cs

```diff
@@ -1,117 +0,0 @@
-// -----------------------------------------------------------------------
-//  <copyright file="TransformWithConversionListener.cs" company="Hibernating Rhinos LTD">
-//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
-//  </copyright>
-// -----------------------------------------------------------------------
-
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.MailingList
-{
-    public class TransformWithConversionListener : RavenTestBase
-    {
-        private class Item
-        {
-            public string Id { get; set; }
-
-            public string Name { get; set; }
-        }
-
-        private class TransformedItem
-        {
-            public bool Transformed { get; set; }
-            public string Name { get; set; }
-            public bool Converted { get; set; }
-        }
-
-        private class EtagTransformer : AbstractTransformerCreationTask<Item>
-        {
-            public EtagTransformer()
-            {
-                TransformResults = items =>
-                                   from item in items
-                                   select new
-                                   {
-                                       item.Name,
-                                       Transformed = true
-                                   };
-            }
-        }
-
-        /*
-        private class DocumentConversionListener  : IDocumentConversionListener
-        {
-            
-            public void BeforeConversionToDocument(string key, object entity, RavenJObject metadata)
-            {
-
-            }
-
-            public void AfterConversionToDocument(string key, object entity, RavenJObject document, RavenJObject metadata)
-            {
-            }
-
-            public void BeforeConversionToEntity(string key, RavenJObject document, RavenJObject metadata)
-            {
-            }
-
-            public void AfterConversionToEntity(string key, RavenJObject document, RavenJObject metadata, object entity)
-            {
-                ((TransformedItem)entity).Converted = true;
-            }
-        }*/
-
-        [Fact(Skip = "RavenDB-6264")]
-        public void SyncApi()
```

Comment:
Restore candidate: we got rid of listeners in favor of events. Should we have events before/after conversion as it was used here?